data/named-accounts.ts

```diff
@@ -293,16 +293,22 @@ const VortexNamedAccounts = {
     targetToken: {
         ...getAddress(mainnet, NATIVE_TOKEN_ADDRESS),
         ...getAddress(coti, '0x639aCc80569c5FC83c6FBf2319A6Cc38bBfe26d1'), // weth
-        ...getAddress(tac, NATIVE_TOKEN_ADDRESS) // tac
+        ...getAddress(tac, NATIVE_TOKEN_ADDRESS), // tac
+        [DeploymentNetwork.HederaTestnet]: NATIVE_TOKEN_ADDRESS,
```

Comment:
first token for swapping, pre-defined and add it to CarbonVortex constructor params when deployment